### ISSUE TITLE
dehydrated v0.6.1 fails to renew certificate

### ISSUE THREAD

**Issue by nsoveiko** [lukas2511/dehydrated#555]
i'm trying to set up dehydrated on a new server and ran into this problem:

the initial request to register and issue a letsencrypt.org staging certificate wen though fine. however, when i try to test the renewal, it fails:
```
# dehydrated -c -x
# INFO: Using main config file /etc/dehydrated/config
# INFO: Using additional config file /etc/dehydrated/config.d/00_gentoo.sh
Processing [***redacted***].org
 + Checking domain name(s) of existing cert... unchanged.
 + Checking expire date of existing cert...
 + Valid till Jul 31 15:57:02 2018 GMT (Longer than 30 days). Ignoring because renew was forced!
 + Signing domains...
 + Generating private key...
 + Generating signing request...
 + Requesting new certificate order from CA...
 + Received 1 authorizations URLs from the CA
 + Handling authorization for [***redacted***].org
 + 1 pending challenge(s)
 + Deploying challenge tokens...
 + Responding to challenge for [***redacted***].org authorization...
  + ERROR: An error occurred while sending post-request to http://[***redacted***].org.org/.well-known/acme-challenge/vZFIDLSvENKWsyVkfzfiEbMi8Gf1ZRtQFFMtGMsHAnM (Status 417)

Details:
HTTP/1.1 417 Expectation Failed
Content-Type: text/html
Content-Length: 363
Connection: close
Date: Wed, 02 May 2018 17:28:51 GMT
Server: httpd/42

<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
         "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
  <title>417 - Expectation Failed</title>
 </head>
 <body>
  <h1>417 - Expectation Failed</h1>
 </body>
</html>
```
request as logged by the server (note the `Expect: 100-continue` part)
```
2018-05-02 17:28:51: (request.c.436) fd: 9 request-len: 249 \nPOST /.well-known/acme-challenge/vZFIDLSvENKWsyVkfzfiEbMi8Gf1ZRtQFFMtGMsHAnM HTTP/1.1\r\nHost: spook.alerik.org\r\nUser-Agent: dehydr4ted/0.6.1 curl/7.59.0\r\nAccept: */*\r\nContent-Type: application/jose+json\r\nContent-Length: 1214\r\nExpect: 100-continue\r\n\r\n
2018-05-02 17:28:51: (response.c.122) Response-Header: \nHTTP/1.1 417 Expectation Failed\r\nContent-Type: text/html\r\nContent-Length: 363\r\nConnection: close\r\nDate: Wed, 02 May 2018 17:28:51 GMT\r\nServer: httpd/42\r\n\r\n 
```
curl 7.59.0, bash 4.4_p12, lighttpd/1.4.45

is this a bug? am i doing something wrong?

**Comment by lukas2511** [comment]
This was fixed in 0.6.2.

**Comment by nsoveiko** [comment]
aw, thank you! i've requested version bump from gentoo.
